tests/pipelines/consisid/test_consisid.py

```diff
@@ -338,8 +339,8 @@ def test_consisid(self):
 
         prompt = self.prompt
         image = load_image("https://github.com/PKU-YuanGroup/ConsisID/blob/main/asserts/example_images/2.png?raw=true")
-        id_vit_hidden = [torch.ones([1, 2, 2])] * 1
-        id_cond = torch.ones(1, 2)
+        id_vit_hidden = [torch.ones([1, 577, 1024])] * 5
+        id_cond = torch.ones(1, 1280)
```

Comment:
the size of the `id_vit_hidden` and `id_cond` is wrong, both A100 and XPU will report below error

>     def forward(self, input: Tensor) -> Tensor:
>       return F.linear(input, self.weight, self.bias)
>         E       RuntimeError: mat1 and mat2 shapes cannot be multiplied (1x2 and 1280x1024)

I checked the needed size for these 2 tensors and use the correct tensor